### DISCUSSION TITLE
https://huggingface.co/microsoft/table-transformer-structure-recognition  这个模型怎么转换，使用？

### DISCUSSION THREAD

**Discussion by nissansz** [RapidAI/RapidTable#7] [Q&A]
https://huggingface.co/microsoft/table-transformer-structure-recognition  这个模型怎么转换，使用？

**Comment by SWHL** [comment]
按照人家文档使用

**Comment by nissansz** [comment]
怎么弄到rapidtable里使用？

**Reply by SWHL** [reply]
需要尝试转换onnx，整理源码，提PR合并进来才可以使用。
不同算法前后处理不同，不能通用

**Reply by Joker1212** [reply]
可以参考https://github.com/RapidAI/TableStructureRec 的测评，其实deepdocdetection用的就是这个，可能是英文模型的原因，指标挺差的，不如rapidTable的中文模型

**Comment by nissansz** [comment]
https://github.com/whn09/table_structure_recognition

这个模型能转换用到rapidtable里吗？怎么操作？

**Reply by SWHL** [reply]
需要尝试转换onnx，整理源码，提PR合并进来才可以使用。

**Reply by nissansz** [reply]
好的，等你更新

**Reply by SWHL** [reply]
我们暂时没空更新这个哈， 期待你的PR

**Comment by nissansz** [comment]
我都不知道怎么在Win10本地跑它这个模型，不会转换，使用。









--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: SWHL  ***@***.***>
 

 
 发送时间: 2024-09-29 15:44:00
 

 
 收件人:RapidAI/RapidTable ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [RapidAI/RapidTable] https://huggingface.co/microsoft/table-transformer-structure-recognition  这个模型怎么转换，使用？ (Discussion #7)
 





 我们暂时没空更新这个哈， 期待你的PR

**Reply by SWHL** [reply]
那建议去这个项目下问问吧
